packages/react-native-bpk-component-phone-input/stories.js

```diff
@@ -209,10 +222,21 @@ const FullyIntegrated = (props: FullyIntegratedProps) => {
       >
         <View style={{ backgroundColor: modalBackground }}>
           <BpkDialingCodeList
-            dialingCodes={props.codes}
+            dialingCodes={dialingCodes}
             onItemPress={onItemPicked}
             renderFlag={renderFlag}
             selectedId={selectedId}
+            ListHeaderComponent={
+              <BpkSectionListSearchField
+                placeholder="Search countries"
+                onChangeText={onSearch}
+              />
+            }
+            ListEmptyComponent={
+              <BpkSectionListNoResultsText>
+                No countries
+              </BpkSectionListNoResultsText>
+            }
```

Comment:
Honestly it's awesome how like ten lines of code adds search to this component 🎉